### PR TITLE
Fix utils test and make it run from Makefile too

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Merging PR https://github.com/dapr/dapr/pull/2829 since it does not have update permission for maintainers.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: N/A

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by wcs1only — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Review by luckyxiaoqiang — DISMISSED** [review]

**Comment by artursouza** [comment]
@luckyxiaoqiang Please, resolve conflict.

**Comment by daixiang0** [comment]
Is it still active, it seems useful.

**Review by yaron2 — DISMISSED** [review]

**Review by mukundansundar — DISMISSED** [review]

**Review by mukundansundar — COMMENTED** [review]
@artursouza one of the utils test is failing

File: utils/utils.go [hunk 1]
```diff
@@ -69,15 +68,7 @@ func GetKubeClient() *kubernetes.Clientset {
...
-	t := dateTime.UTC()
-
-	year, month, day := t.Date()
-	hour, minute, second := t.Clock()
-	micros := t.Nanosecond() / 1000
-
-	return fmt.Sprintf(
-		"%04d-%02d-%02dT%02d:%02d:%02d.%03dZ",
-		year, month, day, hour, minute, second, micros)
+	return dateTime.UTC().Format("2006-01-02T15:04:05.999999Z")
```
Comment (artursouza) [inline utils/utils.go:71]: This method was actually wrong. I've updated the test scenarios to reproduce a bug exposed via a flaky test (time.Now() makes these type of tests flaky).

**Comment by artursouza** [comment]
@yaron2 Needs approval here again.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**